PCBot/plugins/rockpaperscissors.py

```diff
@@ -86,34 +86,37 @@ class RPSView(miru.View):
     challengee: hikari.User
     challenger_pick: Optional[RPSPick] = None
     challengee_pick: Optional[RPSPick] = None
+    er_pick = 0
+    ee_pick = 0
     
     message_emojis = ['\N{ROCK}', '\N{SCROLL}', '\N{BLACK SCISSORS}']
 
     async def determine_winner(self, ctx: miru.ViewContext) -> None:
         """Determine if the game has finished and if so who won."""
         if self.challenger_pick is None or self.challengee_pick is None:
             return
-        if self.challenger_pick == self.challengee_pick:
+        self.er_pick = ((self.challenger_pick.value) % 3)
+        self.ee_pick = ((self.challengee_pick.value) %3)
+        
+        if self.er_pick == self.ee_pick:
             await ctx.edit_response('It was a tie!')
+        # rock = 0, paper = 1, scissors = 2
         elif ((
-           self.challenger_pick is RPSPick.Paper and
-           self.challengee_pick is RPSPick.Rock
+            self.er_pick == 1 and self.ee_pick == 0
           ) or (
-           self.challenger_pick is RPSPick.Scissors and
-           self.challengee_pick is RPSPick.Paper
+            self.er_pick == 0 and self.ee_pick == 2
           ) or (
-           self.challenger_pick is RPSPick.Rock and
-           self.challengee_pick is RPSPick.Scissors
+            self.er_pick == 2 and self.ee_pick == 1
```

Comment:
What I meant when I suggested this was that it is possible to replace the entire 6 condition if block with just one check. You had the required mod expression in edit_response in your previous pull.